Odata-docs/webapi-8/fundamentals/actions-functions.md

```diff
@@ -0,0 +1,409 @@
+
+## Difference between bound and unbound operations.
+From OData V4 spec, functions and actions can be either bound to a type or unbound.
+Bound operations are bound to an entity type, primitive type, complex type, or a collection.
+Unbound operations are invoked as static operations on the service.
+
+## Practical examples
+We will add detailed examples on how to configure bound and unbound functions and actions.
+We will start by setting up the OData WebAPI 8.x project.
+
+### CLR classes
+```csharp
+public class Book
+{
+    public string ID { get; set; }
+    public string Title { get; set; }
+    public bool ForKids { get; set; }
+}
+
+public class BookRating
+{
+    public string ID { get; set; }
+    public int Rating { get; set; }
+    public string BookID { get; set; }
+}
+```
+
+### Controller
+```csharp
+public class BooksController : ODataController
+{
+    // Get ~/Books
+    [EnableQuery]
+    public IActionResult Get()
+    {
+        return Ok(DataSource.Instance.Books);
+    }
+}
+```
+
+### Build Edm Model
+```csharp
+private static IEdmModel GetEdmModel()
+{
+    var builder = new ODataConventionModelBuilder();
+    builder.EntitySet<Book>("books");
+    var model = builder.GetEdmModel();
+    return model;
+}
+```
+
+## Bound Function
+Bound functions can be bound to an entity type or a collection of entity type.
+In our example model above, the bound function can be bound to a single book or a collection of books.
+We will demonstrated how to add a function that is bound to a collection.
+
+In the Edm Model, we modify the code as follows.
+
+```csharp
+private static IEdmModel GetEdmModel()
+{
+    var builder = new ODataConventionModelBuilder();
+    builder.EntitySet<Book>("books");
+    builder.EntityType<Book>().Collection
+        .Function("mostRecent")
+        .Returns<string>();
+    var model = builder.GetEdmModel();
+    return model;
+}
+```
+
+In the Edm Model above, we are adding a `mostRecent` function bound to the books collection. The function returns the `ID` of the most recent book. The `ID` is of `string` type.
+
+We will update the `BooksController` as follows.
+
+```csharp
+public class BooksController : ODataController
+{
+    // Get ~/Books
+    [EnableQuery]
+    public IActionResult Get()
+    {
+        return Ok(DataSource.Instance.Books);
+    }
+
+    [HttpGet("odata/Books/MostRecent()")]
+    public IActionResult MostRecent()
+    {
+        var maxBookId = DataSource.Instance.Books.Max(x => x.ID);
+        return Ok(maxBookId);
+    }
+}
+```
+
+If we invoke `GET odata/Books/MostRecent()`
+We get the response below:
+
+```json
+{
+    "@odata.context": "http://localhost:5000/odata/$metadata#Edm.String",
+    "value": "8"
+}
+```
+
+## Unbound function
+Unbound functions don’t bound to any type and they are called as static operations. All unbound function overloads MUST have same return type.
```

Comment:
"they are called as static operations"  => this could be confusing. We can say " they look like a static method in C#" ?